src/api/registration/registration.controller.js

```diff
@@ -17,9 +17,12 @@ const createNewRegistration = async registration => {
   }
 
   // RESOLUTION
+  var moment = require("moment");
+  const reg_submission_date = moment().format("YYYY MM DD");
```

Comment:
Should also add a unit test to make sure this date is being returned - even if coverage is already at 100%!